smif/app/src/components/ConfigForm/NarrativeConfigForm.js

```diff
@@ -16,7 +16,9 @@ class NarrativeConfigForm extends Component {
     }
 
     componentDidMount(){
-        document.addEventListener("keydown", this.handleKeyPress, false)
+        if (Object.keys(document).length) {
```

Comment:
Is this condition used to check for the availability of a document object before interacting with it, and only expected to be false when running tests?

We should probably use enzyme's full rendering mode for components which need to interact with the DOM (see http://airbnb.io/enzyme/docs/api/mount.html) rather than changing code to work in the shallow test environment.

In any case, I'd expect `if (document !== undefined)` to do the job.